libs/portal/shared/features/form/ui/combobox-field.tsx

```
@@ -89,24 +90,27 @@
-                    className="border border-border backdrop-blur-md bg-card hover:bg-white/10 justify-start max-w-inherit p-1.5 gap-1.5 h-auto"
-                  >
-                    <img src={chevronUpDown} className="size-6" />
-                    <span
-                      className={cn(
-                        'truncate',
-                        (!field.value || field.value === 'Search') && 'text-muted text-base font-light italic',
-                      )}
+                  <FormControl>
+                    <Button
+                      variant="outline"
+                      role="combobox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)